src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -45,8 +51,8 @@
                             </ul>
                             <div class="tab-content">
                                 {% for code, form_tab in admin.formtabs %}
-                                    {% set _tab_name = 'tab_'~admin.uniqid~'_'~loop.index %}
-                                    <div class="tab-pane fade{% if (not app.request.query.has('_tab') and loop.first) or (app.request.query.get('_tab') == _tab_name) %} in active{% endif %}" id="{{ _tab_name }}">
+                                    {% set _tab_name = 'tab_'~admin.uniqid~'_'~random_string~'_'~loop.index %}
+                                    <div class="tab-pane fade{% if (not app.request.query.has('_tab') and loop.first) or (tab_query_index == loop.index) %} in active{% endif %}" id="{{ _tab_name }}">
```

Comment:
This line is too long

Reply:
Maybe we need  a condition variable to let the line become shorter

Reply:
Just use linebreaks
```
<div 
    class="thing"
    id="other_thing"
>
```